lib/ws_servers/api/metrics_client.js

```
@@ -0,0 +1,127 @@
+'use strict'
+
+const WebSocket = require('ws')
+const flatPromise = require('flat-promise')
+const { RESTv2 } = require('bfx-api-node-rest')
+
+const maxWaitTime = 2 * 1000
+const pingInterval = 20 * 1000
+const delayReconnect = 1000
+const tokenTTL = 60 * 60
+
+const DEFAULT_METRICS_SERVER_URL = 'ws://localhost:9090/ws/'
```

I rather throw an exception than use a default localhost URL